checker/src/org/checkerframework/checker/units/UnitsAnnotatedTypeFactory.java

```diff
@@ -62,7 +62,7 @@ public UnitsAnnotatedTypeFactory(BaseTypeChecker checker) {
         addTypeNameImplicit(java.lang.Void.class, BOTTOM);
     }
 
-    protected Map<String, UnitsRelations> getUnitsRel() {
+    private Map<String, UnitsRelations> getUnitsRel() {
```

Comment:
Why make the method private? Is there any harm in letting a subclass use/override this method?